CTP/isolation/src/com/navercorp/cubridqa/isolation/impl/FeedbackDB.java

```diff
@@ -68,7 +68,7 @@ public void onTaskStartEvent(String buildFilename) {
 
 		Timestamp d = new Timestamp(System.currentTimeMillis());
 
-		String category = context.getProperty("main.testing.category");
```

Comment:
as is:
String category = context.getProperty("main.testing.category", "isolation");

to be:
String Context.getTestCategory() {
    return getProperty("main.testing.category", "isolation");
}